handbook/product/beta_and_experimental_feature_labels.md

```diff
@@ -0,0 +1,61 @@
+# Beta and experimental feature labels
+
+We use a hierarchy of feature labels to set expectations with our users and internally: `Experimental` << `Beta` << GA. 
```

Comment:
nit (non-blocking): The arrow directions really messed with my head.

```suggestion
We use a hierarchy of feature labels to set expectations with our users and internally: `Experimental` ➡️ `Beta` ➡️ GA. 
```

Reply:
+1 to this small change

Reply:
Haha fair point.

Reply:
+++ to this change